tests/test_constraints.py

```diff
@@ -35,6 +35,9 @@ def mock_uv_available(self, monkeypatch, uv_available_value):
             "dallinger.constraints.uv_available", lambda: uv_available_value
         )
 
+    @pytest.mark.skip(
```

Comment:
This note is too vague... But also, I'm not sure the problem is still occurring? I tried reverting the commit in #8300, and I don't see the error, though there are some other problems which look unrelated. I pinned pip to fix one of them.

Reply:
We have this error in all CI runs which are unrelated to this PR currently:
```
FAILED demos/tests/test_bartlett.py::TestBartlett1932::test_bartlett_selenium[chrome_headless] - urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='localhost', port=52467): Read timed out. (read timeout=120)
= 1 failed, 1042 passed, 58 skipped, 6 xfailed, 38 warnings, 2 rerun in 962.86s (0:16:02) =
tests: exit 1 (978.86 seconds) /home/runner/work/Dallinger/Dallinger> coverage run /home/runner/work/Dallinger/Dallinger/.tox/tests/bin/pytest tests demos --runslow --chrome-headless --prolific --prolific_writes pid=6652
```
It has nothing to do with this PR, so I will merge as suggested :+1: